### DISCUSSION TITLE
How to implement this cursor？这个光标怎么实现的？

### DISCUSSION THREAD

**Discussion by JIAFENG123** [editablejs/editable#172] [Q&A] [answered]
大佬们，这个编辑器很牛啊，我想请教下这个编辑器光标怎么实现的, 感谢大佬们
Handsome guy, this editor is awesome. I would like to know how to implement the cursor in this editor. Please give me some advice. thanks!

**Comment by big-camel** [comment, marked as answer]
使用 div 模拟光标的显示，然后再隐藏一个 textarea，需要输入的时候使用 textarea 接收输入的文本
